distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc

```diff
@@ -62,4 +62,10 @@ a|Update the `<${branding}_HOME>/etc/org.ops4j.pax.web.cfg` file to add the entr
 Setting this configuration may break compatibility to legacy systems that do not support two-way SSL.
 ====
 
+|Session Inactivity Timeout
+|unauthorized access
+a|Update the Session configuration to have no greater than 10 minute Session Timeout.
+
+ * To set the timeout, navigate to ${ddf-security}
```

Comment:
Can you revise the steps to
`* Navigate to the *${admin-console}.`
`* Select the *${ddf-security}* application.`
`* Select the *Configuration* tab.
`* Select *Session*.`
`* Set *Session Timeout (in minutes)* to `10` (or less).`